src/torchcodec/_core/CpuDeviceInterface.cpp

```diff
@@ -80,7 +69,10 @@ void CpuDeviceInterface::initializeVideo(
     first = false;
   }
   if (!transforms.empty()) {
-    filters_ = filters.str();
+    // Note that we ensure that the transforms come AFTER the format conversion.
+    // This means that the transforms are applied in the output pixel format and
+    // colorspace.
+    filters_ += "," + filters.str();
```

Comment:
Below I'm suggesting that maybe we could leave the default of `filters_` to "copy", because I didn't understand this comment at first. I think the comment is only clear if one knows that the default of `filters_` is the format conversion. If we keep `copy` as the default, then this becomes


```suggestion
    filters_ = "format=rgb24" + filters.str();
```

and this comment is more obvious.

I don't have a strong opinion and if I'm not making any sense, please feel free to ignore (it's the end of the day!)